chaudloader/src/hooks/stage1.rs

```diff
@@ -165,7 +169,53 @@ unsafe fn on_pck_load(
                         .chain(std::iter::once(0))
                         .collect::<Vec<_>>();
                     let mod_pck_wstr_ptr = mod_pck_wstr.as_ptr();
-                    LoadFilePackage.call(sound_engine_class, mod_pck_wstr_ptr, out_pck_id);
+                    let load_mod_pck_res =
+                        LoadFilePackage.call(sound_engine_class, mod_pck_wstr_ptr, out_pck_id);
+                    log::info!(
+                        "{} LoadFilePackage Result: {}",
+                        pck.to_str().unwrap(),
+                        load_mod_pck_res
+                    );
+                }
+            }
+        }
+        _ => (),
+    }
+    return return_val;
+}
+
+unsafe fn on_bnk_load(
+    bnk_file_name: *const winapi::shared::ntdef::CHAR,
+    out_bnk_id: *mut u32,
+) -> i32 {
+    let bnk_str = std::ffi::CStr::from_ptr(bnk_file_name)
+        .to_string_lossy()
+        .to_string();
+
+    let return_val = LoadBank.call(bnk_file_name, out_bnk_id);
+    match bnk_str.as_str() {
+        "Vol1Global.bnk" | "Vol2Global.bnk" => {
+            // Only initialize this once in case both Vol1Global.bnk and Vol2Global.bnk is loaded
```

Comment:
```rust
// Only initialize this once in case Vol1Global.bnk or Vol2Global.bnk is loaded
```

Reply:
The reason for this check is to only load new bnks/pcks once in case the game tries to load both of the files being checked.  This would be the case if the user was using the randomized battle music mod

Reply:
Ok, I had no idea that's what you were referring to. The current wording reads to me as "initialize once both Vol1 and Vol2 are loaded" but what you actually mean is "if both Vol1 and Vol2 are loaded, don't initialize again".

Changing the wording from "both ... and" to "or" makes the "when will this be initialized" part more explicit while not changing anything about the "only initialized once" part (in case both get loaded).

If you specifically want to convey something about the randomized battle music mod then please write that in the comment.

Btw, Vol1Global.bnk and Vol2Global.bnk are both reserved filenames. How would the randomized battle music mod deal with that restriction? I assume it would simply want to load the .bnk file directly from the other game's install folder right?